app/models/concerns/smooch_search.rb

```diff
@@ -323,8 +323,9 @@ def send_explainer_to_user(ex_item_id, tipline_request, report)
       self.get_installation(self.installation_setting_id_keys, data['app_id']) if self.config.blank?
       no_body = (tipline_request.platform == 'Facebook Messenger' && !report.url.blank?)
       Rails.logger.info "[Smooch Bot] Sending explainer report to user #{uid} for ExplainerItem with ID #{ex_item_id}..."
-      last_smooch_response = self.send_message_to_user(uid, report.text(nil, no_body), {}, false, true, 'explainer_report')
-      self.save_smooch_response(last_smooch_response, tipline_request.associated, data['received'], 'fact_check_report', tipline_request.language)
+      last_smooch_response = self.send_message_to_user(uid, report.text(nil, no_body), {}, false, true, 'report')
+      # FIXME: We should rename the WhatsApp template "fact_check_report" to "article_report"
+      self.save_smooch_response(last_smooch_response, tipline_request.associated, data['received'], 'fact_check_report', tipline_request.language, { explainer_id: ex_item_id })
```

Comment:
ex_item_id is ExplainerItemId not Explainer id so I pushed a new commit to fix this one